content/en/docs/Monitoring Kyverno With Prometheus Metrics/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Monitoring Kyverno
+description: Monitor the activities associated with the Kyverno policies applied over your cluster with a good set of Prometheus-compliant metrics
+weight: 45
+---
+
+## Introduction
+
+As a cluster admistrator or just a normal dev associated with a Kubernetes cluster, it can certainly prove to be beneficial for you to have capabilities around monitoring the state of the Kyverno policies applied over your cluster. Things like tracking the policies, the changes associated with them, the activity associated with the incoming requests hitting them and the results associated with them can prove to be extremely useful as a part of cluster-observability compliance.
+
+Alongside this, providing a non-rigid granularity of monitoring the above targets from rule's level to policy's level to entire admission request's level would give you enough flexibility to extract a diversified set of insights from these metrics depending on your need or/and SLA requirements.
+
+## Installation and Setup
+
+Whenever you install Kyverno via helm, a service called `kyverno-svc` gets created as well inside the `kyverno` namespace and this service ends up exposing the metrics at its port no. 8000.
+
+```sh
+$ values.yaml
+
+...
+service:
+  port: 443
+  type: ClusterIP
+  # Only used if service.type is NodePort
+  nodePort:
+  ## Kyverno's metrics server will be exposed at this port
+  metricsPort: 8000
+  ## The Node's port which will allow access Kyverno's metrics at the host level. Only used if service.type is NodePort.
+  metricsNodePort: 8000
```

Comment:
Lets remove this, as Kyverno does not use a node port

Reply:
So, the helm chart is configured in such a way that Kyverno can expose itself as a clusterIP (default and private), NodePort (publicly exposed via a host's port) and LoadBalancer (most generically used - publicly exposed via a cloud-provisioned LoadBalancer).
For more details, please checkout the following comment. It has some important details around this.

Reply:
Ok, got it.